tutorials/noisepy_pnwstore_tutorial.ipynb

```diff
@@ -355,9 +378,9 @@
   },
   "gpuClass": "standard",
   "kernelspec": {
-   "display_name": ".env",
```

Comment:
using `.env` is a convention for placing environments within a repo. It's part of python's standard `.gitignore`:

https://github.com/github/gitignore/blob/4488915eec0b3a45b5c63ead28f286819c0917de/Python.gitignore#L123